events/events-team/marketing/README.md

```diff
@@ -11,78 +11,110 @@ Contributor Summit.
 
 ## Overview
 
-As Marketing Lead, you are responsible for the overall marketing communications
-timeline. The role also includes ensuring all non-technical content (web, mail,
+As Communications Lead, you are responsible for the overall marketing communications 
```

Comment:
Extra trailing space - only if you're updating for the other spelling issue... ;)

Reply:
resolved :)